src/components/Animals/Animals.js

```diff
@@ -0,0 +1,48 @@
+import React, { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import Dropdown from './Dropdown';
+import Animal from './Animal';
+import './Animal.css';
+import { fetchAnimalsDataFromApi } from '../../redux/animalia/animalia';
+
+function Animals() {
+// get Animals data from the store
+  const animals = useSelector((state) => state.animals);
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    if (!animals.length) {
+      dispatch(fetchAnimalsDataFromApi());
+    }
+  }, [animals.length, dispatch]);
+  return (
+    <div>
+      <Dropdown />
+      <div className="animal-wrapper">
+        {
+          animals.map((animal) => (
+            <Animal
+              key={animal.id}
+              id={animal.id}
+              image={animal.image_link}
+              name={animal.name}
+              type={animal.animal_type}
+              latinName={animal.latin_name}
+              activeTime={animal.active_time}
+              minLength={animal.length_min}
+              maxLength={animal.length_max}
+              minWeight={animal.weight_min}
+              maxWeight={animal.weight_max}
+              lifeSpan={animal.lifespan}
+              habitats={animal.habitat}
+              diet={animal.diet}
+              locations={animal.geo_range}
+            />
+          ))
+        }
+      </div>
+    </div>
+  );
```

Comment:
- [ ] Your home page should contain a search functionality for filtering items on the home page. Kindly implement this

  ![gim](https://user-images.githubusercontent.com/54848363/189332404-dcdcd43e-dd5c-4a77-afe8-b081a9882b6d.jpg)

Reply:
> * [ ]  Your home page should contain a search functionality for filtering items on the home page. Kindly implement this
>   ![gim](https://user-images.githubusercontent.com/54848363/189332404-dcdcd43e-dd5c-4a77-afe8-b081a9882b6d.jpg)

- [ ] As requested by the previous reviewer, please make sure the Home page has a search functionality for filtering your animal list using given parameters, e.g. animal. name.